README.md

```diff
@@ -44,8 +44,21 @@ $ brew install dnsmasq
 $ sudo brew services start dnsmasq
 ```
 
+### What services might you have available? Let's find out! ###
```

Comment:
Maybe just `What services have you installed with 'brew services'? Let's find out!`?